simulation.py

```diff
@@ -0,0 +1,92 @@
+"""
+Fuctions for a process to simulate an alpha.
+"""
+
+import argparse
+import copy
+import importlib
+from thousandaire.constants import DATA_LIST_ALL, TRADING_INSTRUMENTS
+from thousandaire.constants import TRADING_CONFIGS, TRADING_REGIONS
+from thousandaire.data_loader import DataLoader
+from thousandaire.simulator import Simulator
+
+PRICE_DATASET = 'price_dataset'
+COST_FUNCTION = 'cost_function'
+
+def initialize():
+    """
+    Initialize data we get.
```

Comment:
Load all available dataset and bind them with workdays.

Return bound data, which is a dict of str (dataset names) -> Dataset.